pkg/volume/glusterfs/glusterfs.go

```diff
@@ -490,13 +495,15 @@ func (r *glusterfsVolumeProvisioner) Provision() (*api.PersistentVolume, error)
 	if len(pv.Spec.AccessModes) == 0 {
 		pv.Spec.AccessModes = r.plugin.GetAccessModes()
 	}
+	sGid := strconv.FormatInt(reqGid, 10)
+	pv.Annotations = map[string]string{"pv.beta.kubernetes.io/gid": sGid}
```

Comment:
oh, missed his comment somehow. And misunderstood this whole thread I think. 

This PR isn't adding a new beta field annotation to act upon, this annotation already has a meaning and was  implemented like this before the linked issues were discussed